src/Cardano/Wallet/Api/Types/WalletDelegation.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+module Cardano.Wallet.Api.Types.WalletDelegation where
+
+import Prelude
+
+import Cardano.Wallet.Api.JSON
+    ( defaultRecordTypeOptions )
+import Data.Aeson
+    ( FromJSON (..), ToJSON (..), genericParseJSON, genericToJSON )
+import GHC.Generics
+    ( Generic )
+
+import qualified Cardano.Wallet.Api.Types.WalletDelegationStatus as T
+
+newtype WalletDelegation = WalletDelegation
+    { _status :: T.WalletDelegationStatus
+    } deriving (Eq, Generic, Show)
```

Comment:
Is this going to have the `target` property, or will we do that later?

Reply:
Well spotted. I'll add this in.